phalcon/Html/TagLocatorFactory.zep

```diff
@@ -87,16 +87,14 @@ class TagLocatorFactory implements LocatorFactoryInterface
      */
     private function getEscaper() -> <EscaperInterface>
     {
-        var container, escaper;
+        var container;
 
         let container = Di::getDefault();
 
-        if (true !== container->has("escaper")) {
-            let escaper = container->getService("escaper");
-        } else {
-            let escaper = new Escaper();
+        if !container->has("escaper") {
```

Comment:
@SidRoberts It seems `Di::getDefault` can return null. What do you think about this?

Reply:
I don't like it. :stuck_out_tongue_closed_eyes: I didn't even realise.

Personally, I think it should behave like this: 

```zephir
public static function getDefault() -> <DiInterface>
{
    if self::_default === null {
        let self::_default = new self();
    }

    return self::_default;
}
```

Reply:
:+1: